doc/source/notebooks/intro_to_gpflow2.pct.py

```diff
@@ -361,4 +364,47 @@ def checkpointing_training_loop(model: gpflow.models.SVGP,
 
 # %%
 params = gpflow.utilities.parameter_dict(model)
-gpflow.utilities.multiple_assign(model, params)
+gpflow.utilities.multiple_assign(model, params)\
+
+# %% [markdown]
+# ### TensorFlow saved_model
+#
+# At present tensorflow does not support saving custom variables like instances of the `gpflow.base.Parameter` class, see [here](https://github.com/tensorflow/tensorflow/issues/34908).
+#
+# However, once trained completed, it is possible to clone the model and replace all Parameters with constants holding the same value:
```

Comment:
```suggestion
# However, once trained completed, it is possible to clone the model and replace all `gpflow.base.Parameter`s with `tf.constant`s holding the same value:
```

Reply:
@vdutor do you mean `tf.Tensor`s? `tf.constant`'s a function not a type right?

Reply:
it is not a type, but it is a by-product of tf.constant that has a different meaning in TF engine.